include/domain_bridge/process_cmd_line_arguments.hpp

```diff
@@ -52,6 +52,12 @@ print_help()
     "                             This overrides any mode set in the YAML file." << std::endl <<
     "                             Defaults to '" << kNormalModeStr << "'." <<
     std::endl <<
+    "    --wait-for-subscription  Will wait for an available subscription before"
+    " bridging a topic." <<
+    std::endl <<
+    "    --dont-wait-for-publisher  Will not wait for an available subscription before"
```

Comment:
A few things here:

- I wonder if we really need to add CLI overrides for these options. @mallanmba, are these useful for you?
- nitpick: it's odd to me to see the contraction "don't" in an option. I would prefer "--do-not" or "--no".
- This option is not implemented. 
-  We should clearly document that the CLI args are overriding any values in the YAML config.
- In case the defaults change in the future, it would be nice to have a way to invert the behavior on the CLI. E.g. have mutually exclusive options: `--wait-for-publisher` and `--no-wait-for-publisher`, or a single option that accepts a bool `--wait-for-publisher true|false`

```suggestion
    "    --do-not-wait-for-publisher  Will not wait for an available publisher before"
```

Reply:
> I wonder if we really need to add CLI overrides for these options. @mallanmba, are these useful for you?

The `--wait-for-subscription` option is certainly useful.
I'm not sure about the `--no-wait-for-publisher` option.

> nitpick: it's odd to me to see the contraction "don't" in an option. I would prefer "--do-not" or "--no".

done in https://github.com/ros2/domain_bridge/pull/52/commits/f29ee71e2b8980e7ce36ce34586f80d9a5aee356.

> This option is not implemented.

:joy:, done in https://github.com/ros2/domain_bridge/pull/52/commits/f29ee71e2b8980e7ce36ce34586f80d9a5aee356.

> We should clearly document that the CLI args are overriding any values in the YAML config.

I haven't added a YAML config for this options yet, but yes we should document that.

Reply:
> In case the defaults change in the future, it would be nice to have a way to invert the behavior on the CLI. E.g. have mutually exclusive options: --wait-for-publisher and --no-wait-for-publisher, or a single option that accepts a bool --wait-for-publisher true|false

Applied the second alternative in https://github.com/ros2/domain_bridge/pull/52/commits/2469cdccc5fb5c4c416ff3fa97be3742c1daba6a

Reply:
Fixed a bug in https://github.com/ros2/domain_bridge/pull/52/commits/f792f5dfd3fb02465379897277475fb9610687d0